app/service/data_svc.py

```diff
@@ -135,6 +135,7 @@ async def _load_abilities(self, plugin):
                     privilege = ab.pop('privilege', None)
                     repeatable = ab.pop('repeatable', False)
                     requirements = ab.pop('requirements', [])
+                    hidden = ab.pop('hidden', False)
```

Comment:
Move the your _load_abilities changes to the new load_ability_file function found on the master branch. That will resolve the merge conflict.